kie-user-bom-parent/pom.xml

```diff
@@ -74,6 +88,33 @@
       </snapshots>
     </repository>
   </repositories>
```

Comment:
I would call this an ugly hack. When we release new IP BOM, it takes up to 24 hours to actually get published (synchronized) into Maven Central (yuck). But its available in repository.jboss.org right after it gets released. To avoid waiting before the upgrade, the repo is also here. I agree this is very far from ideal, but I am not aware of any better alternative right now.